modules/compression/compression.c

```diff
@@ -450,6 +463,45 @@ static int fixup_whitelist_free(void **param)
 	return free_whitelist(*param);
 }
 
+static int fixup_mc_compact_flags(void **param)
+{
+	str *s = (str *) *param;
+	int st;
+	int *flags;
+
+	flags = pkg_malloc(sizeof *flags);
```

Comment:
There is no need to allocate here the int for the flags, you can store the value directly over the `void *` (its size is larger or the same as of an `int`). Check how is done here https://github.com/OpenSIPS/opensips/blob/master/modules/drouting/drouting.c#L3870